docs/conf.py

```diff
@@ -29,8 +29,6 @@
 
 import os
 
-import sphinx.environment
```

Comment:
why?

Reply:
`autoflake` found this as unused import.